# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmFrameIter.cpp
# Commit: ce799cb566ac
# Full Hash: ce799cb566ac0fb95468118608132be11db6a359
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-24 21:54:20
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r c10609d434ae -r ce799cb566ac js/src/wasm/WasmFrameIter.cpp
--- a/js/src/wasm/WasmFrameIter.cpp	Thu Mar 24 17:40:32 2022 +0000
+++ b/js/src/wasm/WasmFrameIter.cpp	Thu Mar 24 17:56:50 2022 +0000
@@ -1578,16 +1578,14 @@
       return "call to native table.fill function";
     case SymbolicAddress::ElemDrop:
       return "call to native elem.drop function";
-    case SymbolicAddress::TableGet:
+    case SymbolicAddress::TableGetFunc:
       return "call to native table.get function";
     case SymbolicAddress::TableGrow:
       return "call to native table.grow function";
     case SymbolicAddress::TableInit:
       return "call to native table.init function";
-    case SymbolicAddress::TableSet:
+    case SymbolicAddress::TableSetFunc:
       return "call to native table.set function";
-    case SymbolicAddress::TableSize:
-      return "call to native table.size function";
     case SymbolicAddress::RefFunc:
       return "call to native ref.func function";
     case SymbolicAddress::PreBarrierFiltering: